src/interfaces/OperationTypes.ts

```diff
@@ -20,9 +20,9 @@ export interface IRollSellOpType {
 export interface IExecSmartContractOpType {
     ExecuteSC: {
         data: [number], // vec of bytes to execute
-        max_gas: number, // maximum amount of gas that the execution of the contract is allowed to cost.
-        coins: string, // represent an Amount in coins that are spent by consensus and are available in the execution context of the contract.
+        max_gas: number, // maximum amount of gas that the execution of the contract is allowed to cost
         gas_price: string, // represent an Amount in coins, price per unit of gas that the caller is willing to pay for the execution.
+        datastore: any,
```

Comment:
Can we change the type here to a concrete one ?

Reply:
done